civihr_employee_portal/views/views_export/views_documents.inc

```diff
@@ -351,6 +351,13 @@ $handler->display->display_options['arguments']['target_contact_id']['summary_op
 $handler = $view->new_display('block', 'Documents Dashboard Manager block', 'block_1');
 $handler->display->display_options['defaults']['title'] = FALSE;
 $handler->display->display_options['title'] = 'Documents Manager';
+$handler->display->display_options['defaults']['access'] = FALSE;
+$handler->display->display_options['access']['type'] = 'role';
+$handler->display->display_options['access']['role'] = array(
+  55120974 => '55120974',
+  37588037 => '37588037',
+  57573969 => '57573969',
```

Comment:
It seems like using RIDs in exports is a known issue. There's a whole section about it [here](https://www.drupal.org/docs/7/modules/features/exportables-and-user-role-ids-in-features). Drupal 8 will apparently use the machine name.

The employee portal features use [role export](https://www.drupal.org/project/role_export) which uses a hash of the machine name when exporting. From their page:

> Hashing machine names? That will never be unique, are you out of your mind?
Certainly not. There will not be a large number of roles on a Drupal site (if you have more than 50 you're doing it wrong). So we can assume that the probability of an id clash is very, very, very, very low. If you encounter such a case, please report it in the issue queue.

I tried changing it to use the machine names in the views file but it didn't work.